### DISCUSSION TITLE
请教一下SPI读取次数

### DISCUSSION THREAD

**Discussion by GXzzzzzzzzz1** [peng-zhihui/HelloWord-Keyboard#169] [Q&A]
我看代码读取11个74HC165的时候，SPI读取的次数实际在12次？我自己焊了一个74HC165的时候也发现，需要读两个字节，舍弃第一个字节，从第2个字节开始才是正确的，为什么会出现这种情况？

**Comment by PIA-1323** [comment]
请问这个问题您找到答案了吗
